src/Indice.Features.Cases.App/src/app/app-routing.module.ts

```diff
@@ -25,7 +25,13 @@ const routes: Routes = [
     path: '', canActivate: [AuthGuardService], children: [
       { path: '', redirectTo: 'dashboard', pathMatch: 'full' },
       { path: 'dashboard', component: DashboardComponent },
-      { path: 'cases', component: CasesComponent },
+      {
+        path: 'cases',
+        children: [
+          { path: '', component: CasesComponent },
+          { path: ':caseTypeCode', component: CasesTypeMenuItemsComponent },
```

Comment:
εδώ πρέπει να παράγουμε δυναμικά λίστα απο paths. Υποθέτω πρέπει να εχει γινει η κλήση των caseTypes ωστε να ξέρουμε πόσα υπομενού πρέπει να κάνουμε

Reply:
Αν παμε με λογική behaviour subject, μηπως βοηθήσει και στο άλλο bug ?

δηλαδη default menu = [ 'αρχικη']

if is admin{
 var previous = subject.value
 _menu.next(previous.push('admin menu')
}

μετά οταν γίνει και η 3η κλήση

 var previous = subject.value
 _menu.next(previous.push('dynamic item')